platform/admin/core/admin-core-alertservice/src/main/java/org/codice/ddf/admin/core/alert/service/internal/AlertServiceBean.java

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.core.alert.service.internal;
+
+import static org.apache.commons.lang.Validate.notNull;
+import static org.codice.ddf.admin.core.alert.service.data.internal.Alert.ALERT_EVENT_ALERT_PROPERTY;
+import static org.codice.ddf.admin.core.alert.service.data.internal.Alert.ALERT_EVENT_TOPIC_VALUE;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import javax.annotation.Nonnull;
+import javax.management.InstanceNotFoundException;
+import javax.management.MBeanRegistrationException;
+import javax.management.MBeanServer;
+import javax.management.ObjectName;
+
+import org.codice.ddf.admin.core.alert.service.data.internal.Alert;
+import org.codice.ddf.admin.insecure.defaults.service.InsecureDefaultsService;
+import org.osgi.service.event.Event;
+import org.osgi.service.event.EventHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * <p>
+ * <b> This code is experimental. While this interface is functional and tested, it may change or be
+ * removed in a future version of the library.</b>
+ * </p>
+ * <p>
+ * TODO javadoc
+ */
+public class AlertServiceBean implements AlertServiceBeanMBean, EventHandler {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AlertServiceBean.class);
+
+    protected static final String INSECURE_DEFAULTS_ALERT_TITLE =
+            "The system is insecure because default configuration values are in use.";
+
+    private final MBeanServer mBeanServer;
+
+    private final InsecureDefaultsService insecureDefaultsService = new InsecureDefaultsService();
+
+    private ObjectName objectName;
+
+    private Map<String, Alert> addedAlerts = new HashMap<>();
+
+    public AlertServiceBean(MBeanServer mBeanServer) {
+        notNull(mBeanServer, "mBeanServer may not be null");
+        this.mBeanServer = mBeanServer;
+    }
+
+    public void init() {
+        try {
+            objectName = new ObjectName(
+                    AlertServiceBean.class.getName() + ":service=alert-service");
+            mBeanServer.registerMBean(this, objectName);
+        } catch (Exception e) {
+            LOGGER.error("Could not register AlertServiceMBean.", e);
```

Comment:
Anything at warn or above should include steps that an admin can take to resolve the situation or if there isn't anything they can do to resolve it, they should be informed of the consequences.